### ISSUE TITLE
Birthdays of relatives are shown twice on Dashboard

### ISSUE THREAD

**Issue by jonasvp** [monicahq/monica#1496]
**Describe the bug**

I added my wife as a contact and our children as well. Now I get two birthday reminders on the dashboard for my daughter: once for her as a contact, once as a child of my wife. The reminders should probably check if they're referring to the same contact.

**Screenshots**

![bildschirmfoto von 2018-06-18 09-52-31](https://user-images.githubusercontent.com/117138/41524336-6c1d0d98-72dd-11e8-9002-1b159fb84632.png)

**Which version are you using:**

Hosted version on https://app.monicahq.com

**Comment by asbiin** [comment]
I cannot reproduce, I think this is already fixed. @djaiss any idea?

**Comment by chrisspiegl** [comment]
I have a similar issue, on the persons' profile I see a whole bunch of reminders for birthdays of the relatives and it's confusing. But on the list on the profile, it only shows the message like "Wish happy birthday to Johanna" (which in this case is the daughter of the contact).

I'm not sure how to solve this problem while still keeping the functionality to have these "wish happy birthday to the relatives.

The only idea is that if the relative has a full contact profile it should not display as a relatives birthday. If it does not have that profile and is just the name + birthday + connection status, it can show.

**Comment by jflatz** [comment]
This seems to still be an issue. 
![monicaDuplicateBday](https://user-images.githubusercontent.com/6561797/58098726-2a8b3880-7ba8-11e9-8bfb-155754e4b077.png)

**Comment by jonasvp** [comment]
Yes, I recently received two reminder emails for the same birthday again as well.

**Comment by tbirrell** [comment]
Pursuant to OP, make sure that the Child attached to the Parent is the actual Child Contact. Otherwise the duplication comes form having two entrees for the same person, and the system won't know they are not different people.

**Comment by jonasvp** [comment]
@tbirrell They are definitely they same contact (at least the URL is the same after clicking on them).

**Comment by tbirrell** [comment]
What version are you on @jonasvp?

**Comment by jflatz** [comment]
@tbirrell Same, these are they same contact (URLs are identical.) I'm currently using the monicahq.com hosted option which is latest build. The contact in my screenshot has one related spouse and one related child.

**Comment by barttenbrinke** [comment]
I'm having the same issue. If you disable the reminder on the person, both of them are removed from the dashboard and if I renable it, they both appear again.

**Comment by nixpc** [comment]
I have the same issue except it repeats 5 times. I also have 5 users on my Monica dashboard.

**Comment by Meuschke23** [comment]
Same Issue for me. 2 Users and it repeats 2 times.
